components/display/tm1638.rst

```diff
@@ -126,10 +126,9 @@ The following example creates a typical digital clock with the ``:`` colon flash
             it.strftime("%H%M", id(homeassistant_time).now());
 
 
-current/components/switch/index.rst
 LEDs
 ----------------
-The TM1638 7 Segment Display Keypad & LED Module has 8 LED lights across the top or middle of the display, depending on the board.  These LEDs are exposed as switches (:doc:`Switch </components/switch>`) with the led property set between 0 and 7.  The board has the LEDs labelled 1-8, but the leds use zero based addressing.   The LED labelled "1" is LED 0 in the configuration.
+The TM1638 7 Segment Display Keypad & LED Module has 8 LED lights across the top or middle of the display, depending on the board.  These LEDs are exposed as switches (:doc:`switches </components/switch/index>`) with the led property set between 0 and 7.  The board has the LEDs labelled 1-8, but the leds use zero based addressing.   The LED labelled "1" is LED 0 in the configuration.
```

Comment:
You misunderstood what I meant there.  The doc link replaces the `switches` word and remove the parentheses.  Otherwise, move the doc link to the end like the other section.

Reply:
Yea I saw it, my bad :)

Reply:
Parens removed